test/fail_compilation/reserved_version.d

```diff
@@ -90,6 +90,8 @@ fail_compilation/reserved_version.d(180): Error: version identifier 'D_Version2'
 fail_compilation/reserved_version.d(181): Error: version identifier 'D_NoBoundsChecks' is reserved and cannot be set
 fail_compilation/reserved_version.d(184): Error: version identifier 'all' is reserved and cannot be set
 fail_compilation/reserved_version.d(185): Error: version identifier 'none' is reserved and cannot be set
+fail_compilation/reserved_version.d(186): Error: version identifier 'Playstation' is reserved and cannot be set
+fail_compilation/reserved_version.d(187): Error: version identifier 'Playstation4' is reserved and cannot be set
```

Comment:
You need to change the line number of all of the above messages to account for the fact that the code is moved two lines down. (E.g. the message about DigitalMars should be about line number 98, not 96).